app/controllers/external_users/advocates/claims_controller.rb

```diff
@@ -13,6 +14,12 @@ def create
 
   private
 
+  def set_form_step
```

Comment:
The intent of this method is to ensure the user is always sent to a valid page, rather than getting a 500 `Bad range` error when trying to submit a page for a step that no longer exists.